hms-app/src/components/input-forms/IdeaForm.tsx

```diff
@@ -239,6 +244,12 @@ function IdeaForm(props: IProps) {
     loadAvailableCategories()
   }, [hackathon])
 
+  useEffect(() => {
+    if (props.loaderState) {
+      props.loaderState(loader)
+    }
+  }, [loader])
```

Comment:
not sure about this, feels like the if condition is wrong 🤔

Reply:
should be right
checks if the prop loaderstate exist 
and if so return this function with the actual value of the loader

Reply:
ok so this was just a bit of my misunderstanding of the code :P it works the way it should